src/view/components/footer/Footer.scss

```diff
@@ -2,16 +2,16 @@
     position: fixed;
     bottom: 0;
     width: 95%;
-    height: 70px;
+    height: 150px;
     display: flex;
     align-items: center;
     justify-content: center;
     background-color: var(--statementBackground);
 
     .add-statement-button {
         background: #5899e0;
-        width: 3.125rem;
-        height: 3.125rem;
+        width: 6.125rem;
+        height: 6.125rem;

```

Comment:
Please avoid using fixed sizes. You can have a max size of 6.125 rem, but not the default.